bot.py

```diff
@@ -43,6 +44,53 @@ async def on_member_join(member):
     await member.dm_channel.send(f"Hi {member.name}, welcome to 관전남 월드!")
 
 
+# remove default help command to use our help command
+bot.remove_command("help")
+
+# Custom help command
+@bot.command(name="help", help="This is help command")
+async def help_command(ctx):
+    """Help command outputs description about all the commands"""
+    # Create to display help command in embedded format
```

Comment:
This comment doesn't make any sense

Reply:
I will change the comment to `"""Illustrate how to use the bot and explains all the commands"""`.
How about this?

Reply:
im talking about `Create to display help command in embedded format`

Reply:
I think this comment doesn't necessary. I will remove it.